### DISCUSSION TITLE
TimeAxis range exclude day

### DISCUSSION THREAD

**Discussion by petrov9** [GSI-CS-CO/chart-fx#394] [Q&A] [answered]
I wanna show for example dates in range 13.01.2020 to 16.01.2020, but exclude 14 and 15 days of january. How I can to do it ?

**Comment by wirew0rm** [comment, marked as answer]
Since the time axis is basically a linear axis, it doesn't supports jumps in the time. You could implement your own axis with a custom transform that skips these days. If you look at the parts of the code that implement the logarithmic axis you might get an idea of what would be needed for this. It might upset things like the Zoomer plugin though.

Another option could be to add a chart for every timespan and have only the first one have a y axis. You can then use the y axis of the first chart in all the other charts too. Of course things like zoom will not work and you have to split your dataset manually into your intervals.

**Reply by yezhengli-Mr9** [reply]
> implement your own axis with a custom transform that skips these days

Dear friends, 
any beginner's guide/ tutorial for writing custom x-axis? Guess as simple as scollable intervals https://github.com/fair-acc/chart-fx/issues/294 with adaptive min, max, volume distribution are good enough so far.

Based on [chart-fx](https://github.com/fair-acc/chart-fx) or other libraries ([javafx](https://github.com/openjdk/jfx), etc.) all are welcome~

Best and thanks,
